FiveCalls/FiveCalls/IssueDone.swift

```diff
@@ -81,7 +85,58 @@ struct IssueDone: View {
             AnalyticsManager.shared.trackPageview(path: "/issue/\(issue.slug)/done/")
             
             store.dispatch(action: .FetchStats(issue.id))
+         
+            // will prompt for a rating after hitting done 5 times
+            RatingPromptCounter.increment {
+                guard let currentScene = UIApplication.shared.connectedScenes.first as? UIWindowScene else {
+                    return
+                }
+                
+                SKStoreReviewController.requestReview(in: currentScene)
+            }
+            
+            // unlikely to occur at the same time as the rating prompt counter
```

Comment:
in the worst case, what happens if they both trigger at the same time, stacked alerts?

Reply:
It's a bit jarring, one pops and then the other before you have a chance to react, but you can still interact with both in turn.